cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/Python.kt

```diff
@@ -408,7 +408,7 @@ interface Python {
      * ```
      */
     class ASTImportFrom(pyObject: PyObject) : ASTBASEstmt(pyObject) {
-        val module: String? by lazy { "module" of pyObject }
+        val module: String by lazy { "module" of pyObject }
```

Comment:
`module` is a raw string of the ‘from’ name, without any leading dots, or `None` for statements such as `from . import foo`

https://docs.python.org/3/library/ast.html#ast.ImportFrom

Reply:
Changed it back; `from . import X` will currently produce a problem expression, since we are not handling packages yet. I would propose doing this separately since it opens a can of worms.